identity/src/commonMain/kotlin/com/android/identity/util/BufferUtil.kt

```diff
@@ -0,0 +1,146 @@
+package com.android.identity.util
+
+import kotlinx.io.Buffer
```

Comment:
Why are you adding this for kotlinx.io.Buffer? There's already helper functions there e.g. readShort() / readShortLe() and writeShort() / writeShortLe().

I don't think we want to add anything to Buffer so I think we just want ByteArrayUtil.kt and ByteStringUtil.kt to begin with.